screeps-game-api/src/game.rs

```diff
@@ -174,8 +176,182 @@ pub mod map {
     pub fn is_room_available(room_name: &str) -> bool {
         js_unwrap!(Game.map.isRoomAvailable(@{room_name}))
     }
+
+    /// Implements `Game.map.findExit`.
+    /// 
+    /// Does not yet support callbacks.
+    pub fn find_exit(from_room: Room, to_room: Room) -> Result<Exit, ReturnCode> {
+        let code: i32 = js_unwrap!{Game.map.findExit(@{from_room.name()}, @{to_room.name()})};
+        Exit::try_from(code).map_err(|v| v.try_into().expect("find_exit: Error code not recognized."))
+    }
+
+    // pub fn find_route(from_room: Room, to_room: Room, route_callback: Option<impl Fn(&str, &str) -> u32>) -> !{
+    //     unimplemented!()
+    // }
+}
+
+pub mod market {
+    use std::collections::HashMap;
+    
+    use stdweb::unstable::TryInto;
+
+    use {Room};
+    use constants::{ReturnCode, ResourceType};
+
+    #[repr(u32)]
+    #[derive(Clone, Debug)]
+    pub enum OrderType {
+        Sell = 0,
+        Buy = 1
+    }
+
+    // impl OrderType {
+    //     fn as_string(&self) -> String {
```

Comment:
If you do want to keep this method in, I don't think it would hurt.

If we want to add in a typed `OrderType` to the transactions, the `serde` translation code would have to use strings anyways since `stdweb` turns it into JSON before loading it in.